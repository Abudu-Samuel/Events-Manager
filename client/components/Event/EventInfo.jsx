import React from 'react';


const EventDetails = ({
  event,
  noEvent
}) => (
  <div>
    <div className="space">
      <div className="container" style={{ marginTop: 99 }}>
        <h2 className="mb-5 font-weight-bold  text-center">{event.title}</h2>
        <div className="row">
          <div className="col-xl-6 mb-4">
            <div className="card prev">
              <img className="img-fluid equal" src={event.image} alt="Card image cap" />
            </div>
          </div>
          <div className="col-xl-6 text-justify">
            <div className="row mb-5">
              <div className="col-xl-4">
                <p className="mb-2 font-weight-bold ">{event.title}</p>
              </div>
              <div className="col-xl-4">
                <p className="font-weight-bold">Type: {event.type}</p>
              </div>
              <div className="col-xl-4">
                <h6 className="font-weight-bold">Date: {event.date}</h6>
              </div>
            </div>
            <p className="text-center font-weight-bold">Description</p>
            <p className="font-weight-bold">{event.description}</p>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default EventDetails;
