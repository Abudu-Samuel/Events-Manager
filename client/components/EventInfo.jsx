import React from 'react';


const EventDetails = ({
  event,
  noEvent
}) => (
  <div>
    <div className="space">
      <div className="container">
        <div className="row">
          <div className="col-md-12">
            <h2 className="mb-5 font-weight-bold grey-text text-center">{event.title}</h2>
          </div>
        </div>
        <div className="row">
          <div className="col-xl-4 mb-4">
            <div className="card">
              <img className="img-fluid" src={event.image} alt="Card image cap" />
              <div className="card-body">
                <p className="grey-text">{event.date} <br /> {event.time}</p>
                <h4 className="card-title pt-1 grey-text text-center">{event.title}</h4>
              </div>
            </div>
          </div>
          <div className="col-xl-8 text-justify">
            <h5 className="mb-2 font-weight-bold grey-text">{event.title}</h5>
            <p className="grey-text">{event.description}
            </p>
            <p className="grey-text">Lorem psused do eiusmod tUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo adipisicing elit,sed sed adipmod tempor incididu nsectipsum dolor icing elit, sed do eiusmod tempor incididunt
                        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo adipisicing elit, sed adipioris nisi nsectetur adir adipisicing elit, sed do eiusmod tempor incididu.
            </p>
            <h6 className="grey-text"><i className="fa fa-map-marker grey-text mr-2 fa-2x" /><strong>Ikeja, Lagos</strong></h6>
            <h6 className="grey-text"><i className="fa fa-clock-o grey-text mr-2 fa-2x" /><strong>Starts {event.date} - Ends {event.date}</strong></h6>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default EventDetails;
