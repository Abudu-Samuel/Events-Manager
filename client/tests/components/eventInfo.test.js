import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import EventInfo from '../../components/Event/EventInfo';
import eventMockedData from '../__mocks__/eventMockedData';


const props = {
  event: eventMockedData.eventData
};

describe('<EventInfo />', () => {
  it('should render Footer correctly', () => {
    const wrapper = shallow(<EventInfo {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
