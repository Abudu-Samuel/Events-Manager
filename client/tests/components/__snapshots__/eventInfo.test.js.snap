// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventInfo /> should render Footer correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventDetails
    event={
      Object {
        "date": "2018-12-06",
        "description": "awesome",
        "image": "image.png",
        "title": "evento",
        "type": "wedding",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="space"
      >
        <div
          className="container"
          style={
            Object {
              "marginTop": 99,
            }
          }
        >
          <h2
            className="mb-5 font-weight-bold  text-center"
          >
            evento
          </h2>
          <div
            className="row"
          >
            <div
              className="col-xl-6 mb-4"
            >
              <div
                className="card prev"
              >
                <img
                  className="img-fluid equal"
                  src="image.png"
                />
              </div>
            </div>
            <div
              className="col-xl-6 text-justify"
            >
              <div
                className="row mb-5"
              >
                <div
                  className="col-xl-4"
                >
                  <p
                    className="mb-2 font-weight-bold "
                  >
                    evento
                  </p>
                </div>
                <div
                  className="col-xl-4"
                >
                  <p
                    className="font-weight-bold"
                  >
                    Type: 
                    wedding
                  </p>
                </div>
                <div
                  className="col-xl-4"
                >
                  <h6
                    className="font-weight-bold"
                  >
                    Date: 
                    2018-12-06
                  </h6>
                </div>
              </div>
              <p
                className="text-center font-weight-bold"
              >
                Description
              </p>
              <p
                className="font-weight-bold"
              >
                awesome
              </p>
            </div>
          </div>
        </div>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
          style={
            Object {
              "marginTop": 99,
            }
          }
        >
          <h2
            className="mb-5 font-weight-bold  text-center"
          >
            evento
          </h2>
          <div
            className="row"
          >
            <div
              className="col-xl-6 mb-4"
            >
              <div
                className="card prev"
              >
                <img
                  className="img-fluid equal"
                  src="image.png"
                />
              </div>
            </div>
            <div
              className="col-xl-6 text-justify"
            >
              <div
                className="row mb-5"
              >
                <div
                  className="col-xl-4"
                >
                  <p
                    className="mb-2 font-weight-bold "
                  >
                    evento
                  </p>
                </div>
                <div
                  className="col-xl-4"
                >
                  <p
                    className="font-weight-bold"
                  >
                    Type: 
                    wedding
                  </p>
                </div>
                <div
                  className="col-xl-4"
                >
                  <h6
                    className="font-weight-bold"
                  >
                    Date: 
                    2018-12-06
                  </h6>
                </div>
              </div>
              <p
                className="text-center font-weight-bold"
              >
                Description
              </p>
              <p
                className="font-weight-bold"
              >
                awesome
              </p>
            </div>
          </div>
        </div>,
        "className": "space",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2
              className="mb-5 font-weight-bold  text-center"
            >
              evento
            </h2>,
            <div
              className="row"
            >
              <div
                className="col-xl-6 mb-4"
              >
                <div
                  className="card prev"
                >
                  <img
                    className="img-fluid equal"
                    src="image.png"
                  />
                </div>
              </div>
              <div
                className="col-xl-6 text-justify"
              >
                <div
                  className="row mb-5"
                >
                  <div
                    className="col-xl-4"
                  >
                    <p
                      className="mb-2 font-weight-bold "
                    >
                      evento
                    </p>
                  </div>
                  <div
                    className="col-xl-4"
                  >
                    <p
                      className="font-weight-bold"
                    >
                      Type: 
                      wedding
                    </p>
                  </div>
                  <div
                    className="col-xl-4"
                  >
                    <h6
                      className="font-weight-bold"
                    >
                      Date: 
                      2018-12-06
                    </h6>
                  </div>
                </div>
                <p
                  className="text-center font-weight-bold"
                >
                  Description
                </p>
                <p
                  className="font-weight-bold"
                >
                  awesome
                </p>
              </div>
            </div>,
          ],
          "className": "container",
          "style": Object {
            "marginTop": 99,
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "evento",
              "className": "mb-5 font-weight-bold  text-center",
            },
            "ref": null,
            "rendered": "evento",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-xl-6 mb-4"
                >
                  <div
                    className="card prev"
                  >
                    <img
                      className="img-fluid equal"
                      src="image.png"
                    />
                  </div>
                </div>,
                <div
                  className="col-xl-6 text-justify"
                >
                  <div
                    className="row mb-5"
                  >
                    <div
                      className="col-xl-4"
                    >
                      <p
                        className="mb-2 font-weight-bold "
                      >
                        evento
                      </p>
                    </div>
                    <div
                      className="col-xl-4"
                    >
                      <p
                        className="font-weight-bold"
                      >
                        Type: 
                        wedding
                      </p>
                    </div>
                    <div
                      className="col-xl-4"
                    >
                      <h6
                        className="font-weight-bold"
                      >
                        Date: 
                        2018-12-06
                      </h6>
                    </div>
                  </div>
                  <p
                    className="text-center font-weight-bold"
                  >
                    Description
                  </p>
                  <p
                    className="font-weight-bold"
                  >
                    awesome
                  </p>
                </div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="card prev"
                  >
                    <img
                      className="img-fluid equal"
                      src="image.png"
                    />
                  </div>,
                  "className": "col-xl-6 mb-4",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <img
                      className="img-fluid equal"
                      src="image.png"
                    />,
                    "className": "card prev",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "img-fluid equal",
                      "src": "image.png",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  "type": "div",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div
                      className="row mb-5"
                    >
                      <div
                        className="col-xl-4"
                      >
                        <p
                          className="mb-2 font-weight-bold "
                        >
                          evento
                        </p>
                      </div>
                      <div
                        className="col-xl-4"
                      >
                        <p
                          className="font-weight-bold"
                        >
                          Type: 
                          wedding
                        </p>
                      </div>
                      <div
                        className="col-xl-4"
                      >
                        <h6
                          className="font-weight-bold"
                        >
                          Date: 
                          2018-12-06
                        </h6>
                      </div>
                    </div>,
                    <p
                      className="text-center font-weight-bold"
                    >
                      Description
                    </p>,
                    <p
                      className="font-weight-bold"
                    >
                      awesome
                    </p>,
                  ],
                  "className": "col-xl-6 text-justify",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <div
                          className="col-xl-4"
                        >
                          <p
                            className="mb-2 font-weight-bold "
                          >
                            evento
                          </p>
                        </div>,
                        <div
                          className="col-xl-4"
                        >
                          <p
                            className="font-weight-bold"
                          >
                            Type: 
                            wedding
                          </p>
                        </div>,
                        <div
                          className="col-xl-4"
                        >
                          <h6
                            className="font-weight-bold"
                          >
                            Date: 
                            2018-12-06
                          </h6>
                        </div>,
                      ],
                      "className": "row mb-5",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": <p
                            className="mb-2 font-weight-bold "
                          >
                            evento
                          </p>,
                          "className": "col-xl-4",
                        },
                        "ref": null,
                        "rendered": Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "evento",
                            "className": "mb-2 font-weight-bold ",
                          },
                          "ref": null,
                          "rendered": "evento",
                          "type": "p",
                        },
                        "type": "div",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": <p
                            className="font-weight-bold"
                          >
                            Type: 
                            wedding
                          </p>,
                          "className": "col-xl-4",
                        },
                        "ref": null,
                        "rendered": Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": Array [
                              "Type: ",
                              "wedding",
                            ],
                            "className": "font-weight-bold",
                          },
                          "ref": null,
                          "rendered": Array [
                            "Type: ",
                            "wedding",
                          ],
                          "type": "p",
                        },
                        "type": "div",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": <h6
                            className="font-weight-bold"
                          >
                            Date: 
                            2018-12-06
                          </h6>,
                          "className": "col-xl-4",
                        },
                        "ref": null,
                        "rendered": Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": Array [
                              "Date: ",
                              "2018-12-06",
                            ],
                            "className": "font-weight-bold",
                          },
                          "ref": null,
                          "rendered": Array [
                            "Date: ",
                            "2018-12-06",
                          ],
                          "type": "h6",
                        },
                        "type": "div",
                      },
                    ],
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Description",
                      "className": "text-center font-weight-bold",
                    },
                    "ref": null,
                    "rendered": "Description",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "awesome",
                      "className": "font-weight-bold",
                    },
                    "ref": null,
                    "rendered": "awesome",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="space"
        >
          <div
            className="container"
            style={
              Object {
                "marginTop": 99,
              }
            }
          >
            <h2
              className="mb-5 font-weight-bold  text-center"
            >
              evento
            </h2>
            <div
              className="row"
            >
              <div
                className="col-xl-6 mb-4"
              >
                <div
                  className="card prev"
                >
                  <img
                    className="img-fluid equal"
                    src="image.png"
                  />
                </div>
              </div>
              <div
                className="col-xl-6 text-justify"
              >
                <div
                  className="row mb-5"
                >
                  <div
                    className="col-xl-4"
                  >
                    <p
                      className="mb-2 font-weight-bold "
                    >
                      evento
                    </p>
                  </div>
                  <div
                    className="col-xl-4"
                  >
                    <p
                      className="font-weight-bold"
                    >
                      Type: 
                      wedding
                    </p>
                  </div>
                  <div
                    className="col-xl-4"
                  >
                    <h6
                      className="font-weight-bold"
                    >
                      Date: 
                      2018-12-06
                    </h6>
                  </div>
                </div>
                <p
                  className="text-center font-weight-bold"
                >
                  Description
                </p>
                <p
                  className="font-weight-bold"
                >
                  awesome
                </p>
              </div>
            </div>
          </div>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
            style={
              Object {
                "marginTop": 99,
              }
            }
          >
            <h2
              className="mb-5 font-weight-bold  text-center"
            >
              evento
            </h2>
            <div
              className="row"
            >
              <div
                className="col-xl-6 mb-4"
              >
                <div
                  className="card prev"
                >
                  <img
                    className="img-fluid equal"
                    src="image.png"
                  />
                </div>
              </div>
              <div
                className="col-xl-6 text-justify"
              >
                <div
                  className="row mb-5"
                >
                  <div
                    className="col-xl-4"
                  >
                    <p
                      className="mb-2 font-weight-bold "
                    >
                      evento
                    </p>
                  </div>
                  <div
                    className="col-xl-4"
                  >
                    <p
                      className="font-weight-bold"
                    >
                      Type: 
                      wedding
                    </p>
                  </div>
                  <div
                    className="col-xl-4"
                  >
                    <h6
                      className="font-weight-bold"
                    >
                      Date: 
                      2018-12-06
                    </h6>
                  </div>
                </div>
                <p
                  className="text-center font-weight-bold"
                >
                  Description
                </p>
                <p
                  className="font-weight-bold"
                >
                  awesome
                </p>
              </div>
            </div>
          </div>,
          "className": "space",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2
                className="mb-5 font-weight-bold  text-center"
              >
                evento
              </h2>,
              <div
                className="row"
              >
                <div
                  className="col-xl-6 mb-4"
                >
                  <div
                    className="card prev"
                  >
                    <img
                      className="img-fluid equal"
                      src="image.png"
                    />
                  </div>
                </div>
                <div
                  className="col-xl-6 text-justify"
                >
                  <div
                    className="row mb-5"
                  >
                    <div
                      className="col-xl-4"
                    >
                      <p
                        className="mb-2 font-weight-bold "
                      >
                        evento
                      </p>
                    </div>
                    <div
                      className="col-xl-4"
                    >
                      <p
                        className="font-weight-bold"
                      >
                        Type: 
                        wedding
                      </p>
                    </div>
                    <div
                      className="col-xl-4"
                    >
                      <h6
                        className="font-weight-bold"
                      >
                        Date: 
                        2018-12-06
                      </h6>
                    </div>
                  </div>
                  <p
                    className="text-center font-weight-bold"
                  >
                    Description
                  </p>
                  <p
                    className="font-weight-bold"
                  >
                    awesome
                  </p>
                </div>
              </div>,
            ],
            "className": "container",
            "style": Object {
              "marginTop": 99,
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "evento",
                "className": "mb-5 font-weight-bold  text-center",
              },
              "ref": null,
              "rendered": "evento",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="col-xl-6 mb-4"
                  >
                    <div
                      className="card prev"
                    >
                      <img
                        className="img-fluid equal"
                        src="image.png"
                      />
                    </div>
                  </div>,
                  <div
                    className="col-xl-6 text-justify"
                  >
                    <div
                      className="row mb-5"
                    >
                      <div
                        className="col-xl-4"
                      >
                        <p
                          className="mb-2 font-weight-bold "
                        >
                          evento
                        </p>
                      </div>
                      <div
                        className="col-xl-4"
                      >
                        <p
                          className="font-weight-bold"
                        >
                          Type: 
                          wedding
                        </p>
                      </div>
                      <div
                        className="col-xl-4"
                      >
                        <h6
                          className="font-weight-bold"
                        >
                          Date: 
                          2018-12-06
                        </h6>
                      </div>
                    </div>
                    <p
                      className="text-center font-weight-bold"
                    >
                      Description
                    </p>
                    <p
                      className="font-weight-bold"
                    >
                      awesome
                    </p>
                  </div>,
                ],
                "className": "row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <div
                      className="card prev"
                    >
                      <img
                        className="img-fluid equal"
                        src="image.png"
                      />
                    </div>,
                    "className": "col-xl-6 mb-4",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <img
                        className="img-fluid equal"
                        src="image.png"
                      />,
                      "className": "card prev",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "className": "img-fluid equal",
                        "src": "image.png",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    "type": "div",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <div
                        className="row mb-5"
                      >
                        <div
                          className="col-xl-4"
                        >
                          <p
                            className="mb-2 font-weight-bold "
                          >
                            evento
                          </p>
                        </div>
                        <div
                          className="col-xl-4"
                        >
                          <p
                            className="font-weight-bold"
                          >
                            Type: 
                            wedding
                          </p>
                        </div>
                        <div
                          className="col-xl-4"
                        >
                          <h6
                            className="font-weight-bold"
                          >
                            Date: 
                            2018-12-06
                          </h6>
                        </div>
                      </div>,
                      <p
                        className="text-center font-weight-bold"
                      >
                        Description
                      </p>,
                      <p
                        className="font-weight-bold"
                      >
                        awesome
                      </p>,
                    ],
                    "className": "col-xl-6 text-justify",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          <div
                            className="col-xl-4"
                          >
                            <p
                              className="mb-2 font-weight-bold "
                            >
                              evento
                            </p>
                          </div>,
                          <div
                            className="col-xl-4"
                          >
                            <p
                              className="font-weight-bold"
                            >
                              Type: 
                              wedding
                            </p>
                          </div>,
                          <div
                            className="col-xl-4"
                          >
                            <h6
                              className="font-weight-bold"
                            >
                              Date: 
                              2018-12-06
                            </h6>
                          </div>,
                        ],
                        "className": "row mb-5",
                      },
                      "ref": null,
                      "rendered": Array [
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": <p
                              className="mb-2 font-weight-bold "
                            >
                              evento
                            </p>,
                            "className": "col-xl-4",
                          },
                          "ref": null,
                          "rendered": Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {
                              "children": "evento",
                              "className": "mb-2 font-weight-bold ",
                            },
                            "ref": null,
                            "rendered": "evento",
                            "type": "p",
                          },
                          "type": "div",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": <p
                              className="font-weight-bold"
                            >
                              Type: 
                              wedding
                            </p>,
                            "className": "col-xl-4",
                          },
                          "ref": null,
                          "rendered": Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {
                              "children": Array [
                                "Type: ",
                                "wedding",
                              ],
                              "className": "font-weight-bold",
                            },
                            "ref": null,
                            "rendered": Array [
                              "Type: ",
                              "wedding",
                            ],
                            "type": "p",
                          },
                          "type": "div",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": <h6
                              className="font-weight-bold"
                            >
                              Date: 
                              2018-12-06
                            </h6>,
                            "className": "col-xl-4",
                          },
                          "ref": null,
                          "rendered": Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "host",
                            "props": Object {
                              "children": Array [
                                "Date: ",
                                "2018-12-06",
                              ],
                              "className": "font-weight-bold",
                            },
                            "ref": null,
                            "rendered": Array [
                              "Date: ",
                              "2018-12-06",
                            ],
                            "type": "h6",
                          },
                          "type": "div",
                        },
                      ],
                      "type": "div",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Description",
                        "className": "text-center font-weight-bold",
                      },
                      "ref": null,
                      "rendered": "Description",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "awesome",
                        "className": "font-weight-bold",
                      },
                      "ref": null,
                      "rendered": "awesome",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
